haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
+HOMEPAGE="https://sourceforge.net/projects/eventual/"
+COPYRIGHT="2011 Alexey Burshtein \
+	   2001-2002 Kevin H. Patterson"
+LICENSE="MIT
+	 BSD (3-clause)"
```

Comment:
Oops, there are a three unwanted spaces in line 6 and one in line 8. (We like to have only one tab and no spaces in there.)
```diff
 COPYRIGHT="2011 Alexey Burshtein \
-	   2001-2002 Kevin H. Patterson"
+	2001-2002 Kevin H. Patterson"
 LICENSE="MIT
-	 BSD (3-clause)"
+	BSD (3-clause)"
```